infrastructure/src/main/java/org/corfudb/infrastructure/BatchProcessor.java

```diff
@@ -70,6 +78,16 @@ public BatchProcessor(StreamLog streamLog, long sealEpoch, boolean sync) {
         this.streamLog = streamLog;
         operationsQueue = new LinkedBlockingQueue<>();
         processorService.submit(this::processor);
+
+        writeRecordTimer = MeterRegistryProvider.getInstance().map(registry ->
```

Comment:
Having a histogram on the batch size/queue size can help answer questions about queuing delay.

Reply:
Added